libafl_frida/src/helper.rs

```diff
@@ -62,13 +62,12 @@ enum CmplogOperandType {
     Mem(capstone::RegId, capstone::RegId, i32, u32),
 }
 
-#[cfg(all(feature = "cmplog", target_arch = "aarch64"))]
 enum SpecialCmpLogCase {
     Tbz,
     Tbnz,
 }
 
-#[cfg(target_vendor = "apple")]
+#[cfg(any(target_os = "macos", target_os = "ios"))]
```

Comment:
Why not keep `target_vendor` here? Should also work on... ipad os or whatever, right?

Reply:
yep, you are correct.